tasks/internal/request-and-upload-signature/tests/mocks.sh

```diff
@@ -16,29 +16,20 @@ function ssh() {
     echo "$ssh_call_count" > "$(workspaces.data.path)/ssh_calls.txt"
 }
 
-function request-signature() {
-  echo Mock request-signature called with: $*
-  echo $* >> $(workspaces.data.path)/mock_request-signature.txt
+function pubtools-sign-msg-container-sign() {
+  >&2 echo "Mock pubtools-sign-msg-container-sign called with: $*"
+  echo "$*" >> "/tmp//mock_pubtools-sign.txt"
+  out=$(sed -n '1p' "$(workspaces.data.path)/mocked_signing_response")
+  sed -i '1d' "$(workspaces.data.path)/mocked_signing_response"
```

Comment:
So here the idea is to print one mocked response at a time, allowing multiple to be stored in the file, right? But I don't see any test that would prepare multiple response. Shouldn't there be one that has at least two responses? Or else why is this mechanism even here?

Reply:
I just copied it form rh-sign-image-cosign mocks. It works for all use cases (one response, multiple different responses) so that's why.

Reply:
OK, but if we never need more than 1 here, it should just do `cat "$(workspaces.data.path)/mocked_signing_response"` - no need for the added complexity.